lib/ex_unit/mix.exs

```diff
@@ -24,7 +24,7 @@ defmodule ExUnit.MixProject do
         module_load_timeout: 60000,
         colors: [],
         exclude: [],
-        formatters: [ExUnit.CLIFormatter],
+        formatters: [ExUnit.CLIFormatter, ExUnit.ManifestFormatter],
```

Comment:
One potential problem with this is that users that replace the `CLIFormatter` will probably not get the benefit of the manifest, unless they configured ExUnit in a way that left any other formatters in place.  Given that the `ManifestFormatter` isn't a "normal" formatter (in the sense that it doesn't format test output to STDOUT), I would imagine that most users who replace the `CLIFormatter` would still want this formatter.  Would it make more sense to automatically add `ManifestFormatter` to the `formatters` list somewhere else so users don't accidentally opt out?  Or is there a reason to expect that some users may not want this enabled?

Reply:
I think we should add the ManifestFormatter "internally", similar to how we always add RunnerStats. Maybe we could even merge both, since we only need one formatter for doing all the book keeping.

Reply:
I'll take a look at merging them.  What should we call the merged formatter?

Reply:
For now I went ahead and kept it separate but added it internally like `RunnerStats`.  I wasn't sure what to call the merged formatter, and if we merged them, I thought the tests would seem odd (since there's a set of tests that directly cover the manifest aspect but not the runner stats aspect--at least as far as a I can see!).

If you'd like them merged, let me know what you'd like to call it and if it would be OK to just test the manifest part or there's a better way to go about it.